### PR TITLE
Adds recitation 2 handout as webpage

### PR DESCRIPTION
**Pull request by anudaweerasinghe**
Content moved from [gdoc](https://docs.google.com/document/d/1tGPH2XPLuiTpJCsi_AcwjzByCk2ZZPZyhvFGNX_HA-k/edit#) to markdown to render as a webpage

### CODE REVIEW

File: recitations.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
```
Comment (victorhuangwq) [inline recitations.md:1]: this adds recitations to the sidebar

File: recitations/reci2-angularjs.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+### Task 6: Special styling for completed todo items
+
+**Goal: We want to apply custom styling for completed todo items, so they appear crossed out**
+
+**Steps:**
+
+- A CSS class called `done-true` for completed todo items has been implemented in `style.css`. Make any changes you’d like to this CSS class
+- Apply this class to the todo text item
+  - Change the `<span>` tag used to render {%raw%}`{{todo.text}}`{% endraw %} to `<span class="done-{{todo.done}}">`
```
Comment (victorhuangwq) [inline recitations/reci2-angularjs.md:106]: I had to add {%raw} to escape the {, else it's not rendering

**Review by victorhuangwq — APPROVED** [review]
Some small issues which I pushed to fix.

**Comment by anudaweerasinghe** [comment]
lgtm

**Pull request merged**